src/main/java/com/jmb/springfactory/service/comment/CommentServiceImpl.java

```diff
@@ -0,0 +1,86 @@
+package com.jmb.springfactory.service.comment;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import com.jmb.springfactory.dao.GenericMySQLService;
+import com.jmb.springfactory.exceptions.NotFoundException;
+import com.jmb.springfactory.exceptions.ServiceLayerException;
+import com.jmb.springfactory.model.bo.BusinessObjectBase;
+import com.jmb.springfactory.model.dto.CommentDto;
+import com.jmb.springfactory.model.dto.TaskDto;
+import com.jmb.springfactory.model.entity.Comment;
+import com.jmb.springfactory.service.GenericServiceImpl;
+import com.jmb.springfactory.service.ValidatorService;
+
+import static com.jmb.springfactory.service.UtilsService.exist;
+import com.jmb.springfactory.service.task.TaskService;
+
+@Service
+public class CommentServiceImpl extends GenericServiceImpl<Comment, CommentDto, BusinessObjectBase, Integer> implements 
+  CommentService {
+
+  @Autowired
+  private TaskService taskService;
+  
+  @Autowired
+  @Qualifier("commentValidatorService")
+  private ValidatorService commentValidatorService;
+  
+  @Override
+  public GenericMySQLService<Comment, Integer> genericDao() {
+    return null;
+  }
+
+  @Override
+  public Class<? extends Comment> getClazz() {
+    return Comment.class;
+  }
+
+  @Override
+  public Class<? extends CommentDto> getDtoClazz() {
+    return CommentDto.class;
+  }
+
+  @Override
+  public Class<? extends BusinessObjectBase> getBoClazz() {
+    return BusinessObjectBase.class;
+  }
+  
+  @Override
+  public CommentDto save(Integer idTask, CommentDto commentDto) throws NotFoundException, ServiceLayerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)